tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Summary.feature

```diff
@@ -4,13 +4,18 @@
     So that I can ensure that the information is complete
 
 Background:
+    Given Contacts exist
+        | FirstName | LastName | EmailAddress            | TelephoneNumber |
+        | Fred      | Robinson | Fred.robinson@email.com | 12312543212     |
     Given Orders exist
-        | OrderId    | Description   | OrderStatusId | Created    | LastUpdated | LastUpdatedBy                        | OrganisationId                       |
-        | C000014-01 | A Description | 1             | 05/05/2020 | 09/05/2020  | a11a46f9-ce6f-448a-95c2-fde6e61c804a | 4af62b99-638c-4247-875e-965239cd0c48 |
+        | OrderId    | Description   | OrderStatusId | Created    | OrganisationContactEmail | LastUpdated | LastUpdatedBy                        | OrganisationId                       |
+        | C000014-01 | A Description | 1             | 05/05/2020 |                          | 09/05/2020  | 7b195137-6a59-4854-b118-62b39a3101e | 4af62b99-638c-4247-875e-965239cd0c48 |
+        | C000014-02 | A Description | 1             | 05/05/2020 | Fred.robinson@email.com  | 09/05/2020  | 7b195137-6a59-4854-b118-62b39a3101e | 4af62b99-638c-4247-875e-965239cd0c48 |
     And the user is logged in with the Buyer role for organisation 4af62b99-638c-4247-875e-965239cd0c48
 
+
 @5321
-Scenario: 1. Displaying the order summary, where the order sections are complete
+Scenario: 1. Displaying the order summary, where the order descript section is complete

```

Comment:
Might be worth looking at specflows example, to make the tests more streamlined, an example of this is in the validation specflow tests

Reply:
changed the tests to remove some of the  checks that were not relevant